apps/codeforafrica/src/components/Author/Author.js

```diff
@@ -1,61 +1,45 @@
 import { RichTypography } from "@commons-ui/core";
-import { Box } from "@mui/material";
+import Stack from "@mui/material/Stack";
 import PropTypes from "prop-types";
 import React from "react";
 
 const Author = React.forwardRef(function Author(props, ref) {
-  const { author, profession, ...other } = props;
+  const { name, profession, ...other } = props;
 
   return (
-    <Box
+    <Stack
```

Comment:
Out of curiosity, why switch to `Stack`?

Reply:
Best tool for the job @esirK :
```jsx
  <Box
      sx={{
        display: "flex",
        flexDirection: { xs: "column", md: "row" },
      }}
  >
```

is exactly equal to

```jsx
  <Stack
    direction={{ xs: "column", md: "row" }}
  >
```